src/components/CreateContract/CreateContractForms/user/index.jsx

```diff
@@ -1,94 +1,204 @@
 import React from 'react';
 import { PropTypes as PT } from 'prop-types';
 import container from './container';
-import { Form, Input, Errorbox } from '../../../common';
+import { withFormik } from 'formik';
+
+import { FormStyleContainer } from '../../../common_styled/FormStyleContainer';
 import { FormContainer } from '../styled';
 
+const FormikEnhancer = withFormik({
+
+  displayName: 'Create Contract - User Form',
+
+  // This updates the form when props change.
+  enableReinitialize: true,
+
+  // validateOn
+  mapPropsToValues: () => {
+    return ({
+      firstName: '',
+      lastName: '',
+      selectedUserId: '-1',
+    });
+  },
+
+  // Custom sync validation
+  validate: ({ firstName, lastName, selectedUserId }) => {
+    let errors = {};
+    if (firstName === '') {
```

Comment:
same as above regarding name validation

Reply:
As far as I know, this is the only validation required for firstName and lastName